bin/install.py

```diff
@@ -47,7 +47,7 @@
 NAILGUN_COMMIT = args.nailgun
 
 NAILGUN_CLIENT_URL = "https://raw.githubusercontent.com/scalacenter/nailgun/%s/pynailgun/ng.py" % NAILGUN_COMMIT
-BLOOP_COURSIER_TARGET = join(BLOOP_INSTALLATION_TARGET, "coursier")
+BLOOP_COURSIER_TARGET = join(BLOOP_INSTALLATION_TARGET, "blp-coursier")
```

Comment:
This change will break the homebrew formula: 

https://github.com/scalacenter/bloop/blob/6fa5009c41a5d8e35f927f02685e0e0075bc7009/project/ReleaseUtils.scala#L75

Reply:
Nice catch :+1: